engine/src/main/java/org/terasology/math/Rotation.java

```diff
@@ -218,8 +218,15 @@ public int hashCode() {
      * Returns all possible rotations for each yaw, pitch & roll combination, including duplicates.
      *
      * @return All possible rotations for each yaw, pitch & roll combination.
+     *
+     * @deprecated use `allRotations` instead
      */
+    @Deprecated
     public static Iterable<Rotation> allValues() {
+        return allRotations();
+    }
+
+    public static Iterable<Rotation> allRotations() {
```

Comment:
What's this about - just better naming? Please move/copy the JavaDoc over to the non-deprecated method if you want to leave this change in.

Reply:
This is part of the old PR, it's not used or anything. I thought about removing it but since I'm aiming to replace the other PR I kept everything in. (if I'm not mistaken, it was you who committed this btw :)

Reply:
I must had a reason for that that is long forgotten now 🙄 🙈

Reply:
Let's remove this change if not used...